code/src/components/RadioButton.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+const knowHeadPhones = [
+    "At par with an Audiophile",
+    "Just Enough to get by",
+    "I don't know much"
+]
```

Comment:
Nice way of putting the options into an array. This is something I should consider as well in the future.